src/bindings/python/tests/test_frontend/test_frontendmanager.py

```diff
@@ -24,14 +25,16 @@
         clear_place_stat,
     )
 except Exception:
-    print("No mock frontend available")
     mock_available = False
 
 # FrontEndManager shall be initialized and destroyed after all tests finished
 # This is because destroy of FrontEndManager will unload all plugins, no objects shall exist after this
 fem = FrontEndManager()
 
-mock_needed = pytest.mark.skipif(not mock_available, reason="mock fe is not available")
+mock_needed = pytest.mark.skipif(not mock_available, reason="Mock frontend is not available."
+                                                            f"\nCheck paths in {os.environ['LD_LIBRARY_PATH']=}")
```

Comment:
why not OV_FRONTEND_PATH?

Reply:
Because is not related to this variable.

Reply:
is it about python version of mock FE?

Reply:
Not related to python version.
This is for python imports an `LD_LIBRARY_PATH` and `PYTHONPATH` must point where python libs are installed.